tools/smufl/smufl_fontgen.js

```diff
@@ -44,6 +44,8 @@ function toVFPath(glyph) {
     'y_min': bb.y1,
     'y_max': bb.y2,
     'ha': bb.y2 - bb.y1,
+    'leftSideBearing': glyph.leftSideBearing,
```

Comment:
This will significantly bloat the font files for non-chordsymbol glyphs, maybe we can parameterize the generation?

Also, to keep the files small, can we choose smaller names (e.g., `lsb` instead of `leftSideBearing`? (And add comments expanding on them.)

Reply:
Those 2 lines are a mistake left over from early text font tool.  I can just remove them.  
Someday, I'd like to make a more general container class that can format glyphs and text in various ways, and then we can revisit where that font metrics information lives, but I don't think it's too useful for music engraving.